packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -67,34 +64,14 @@ const SubMenu = ({
       updateTabIndex(activeIndex);
     }
 
-    if (trigger === "click") {
-      if (activeIndex === -1 && !_isOpen && wasPreviouslyOpen) {
-        titleRef.current && titleRef.current.focus();
-      }
-    }
-
-    if (trigger === "hover") {
-      if (
-        activeIndex === -1 &&
-        !_isOpen &&
-        (wasPreviouslyOpenBeforeTimeout || wasPreviouslyOpen)
-      ) {
-        titleRef.current && titleRef.current.focus();
-      }
+    if (activeIndex === -1 && !_isOpen && wasPreviouslyOpen) {
+      titleRef.current && titleRef.current.focus();
     }
 
     if (activeIndex === -1 && _isOpen) {
       menuRef.current && menuRef.current.focus();
     }
-  }, [
-    activeIndex,
-    _isOpen,
-    titleRef,
-    menuRef,
-    trigger,
-    wasPreviouslyOpen,
-    wasPreviouslyOpenBeforeTimeout,
-  ]);
+  }, [activeIndex, _isOpen, titleRef, menuRef, wasPreviouslyOpen]);
```

Comment:
Why was this removed?

Reply:
rebase

Reply:
Rebase or merge now or include those line back bro, they are important.

Reply:
Done 👍